packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts.ejs

```diff
@@ -112,8 +112,11 @@ export class <%= className %>Controller {
       },
     },
   })
-  async findById(@param.path.<%= idType %>('id') id: <%= idType %>): Promise<<%= modelName %>> {
-    return this.<%= repositoryNameCamel %>.findById(id);
+  async findById(
```

Comment:
based on your question in [comment](https://github.com/strongloop/loopback-next/pull/4114#issuecomment-553091517), the response schema should also be updated to 
```
content: {
  `application/json`: {
      schema: getModelSchemaRef(<%= modelName %>, includeRelations),
    }
}
```